py/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Introduces a project scope sampling rule type. ([#5077](https://github.com/getsentry/relay/pull/5077)))
 - Add InstallableBuild and SizeAnalysis data categories. ([#5084](https://github.com/getsentry/relay/pull/5084))
+- Add `retentions` to the project configuration. ([#5135](https://github.com/getsentry/relay/pull/5135))
```

Comment:
You don't need to edit this CHANGELOG file since you didn't make any Python changes.

Reply:
Yeah we need the Python changelog, project configs are (heavily) exposed in the Python library, and before the changes can be used by Sentry a new Python release needs to be cut.